### PR TITLE
DOC/TA-707: Document how to remove navigation if an uninstalled addon

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: docs/Administration.rst [hunk 1]
```diff
@@ -377,6 +380,47 @@ Hover your cursor over Site, then click:
...
+.. _RemovingApp:   
+   
+============================================
+Removing an application from eXo Platform 
+============================================
+
+When installing an :ref:`addon <eXo_addons>` in eXo Platform, an application appears in the platform's navigation
+and it could be added to spaces applications list.
+
+When :ref:`unistalling the addon <AddonsManagement.Uninstalling>`, it's corresponding navigation remains
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
When :ref:`unistalling the addon <AddonsManagement.Uninstalling>`, its corresponding navigation remains
```

File: docs/Administration.rst [hunk 2]
```diff
@@ -377,6 +380,47 @@ Hover your cursor over Site, then click:
...
+    
+    |TA2|
+    
+2-  Right click on the corresponding application's navigation i.e. the navigation of the unistalled addon
+
+3-  Click on ``Delete Node`` button and then the confirmation button.
+
+    |TA3|       
+    
+If the addon belongs to a space's applications list, these are the needed steps and which could be done by the
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
If the application was installed in a space, below are the required steps to be done by the 
```

File: docs/Administration.rst [hunk 3]
```diff
@@ -377,6 +380,47 @@ Hover your cursor over Site, then click:
...
+    |TA2|
+    
+2-  Right click on the corresponding application's navigation i.e. the navigation of the unistalled addon
+
+3-  Click on ``Delete Node`` button and then the confirmation button.
+
+    |TA3|       
+    
+If the addon belongs to a space's applications list, these are the needed steps and which could be done by the 
+space's owner or a spaces manager:
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
space owner or any space manager:
```

File: docs/Administration.rst [hunk 4]
```diff
@@ -377,6 +380,47 @@ Hover your cursor over Site, then click:
...
+
+1-  Go to Space settings |TA4| then to Navigation tab
```
Comment (plamarque) [inline docs/Administration.rst]:
```suggestion
1-  Go to Space Settings |TA4| then to Navigation tab 
```

File: docs/Administration.rst [hunk 5]
```diff
@@ -377,6 +380,47 @@ Hover your cursor over Site, then click:
...
+
+When installing an :ref:`addon <eXo_addons>` in eXo Platform, an application appears in the platform's navigation
+and it could be added to spaces applications list.
+
+When :ref:`unistalling the addon <AddonsManagement.Uninstalling>`, it's corresponding navigation remains
+in the platform.
+
+To completely remove it, follow these steps:
+
+1-  Switch to site's edit mode by clicking on Edit -> Site -> Edit Navigation
```
Comment (ahamdi) [inline docs/Administration.rst:402]: We need to indicate that for eXo platform 6.0, we need to install exo-layout-management to be able to edit site layout/navigation

**Review by plamarque — DISMISSED** [review]
some language improvements suggested

File: docs/Administration.rst (hunk 5 above)
Comment (mzorai) [reply docs/Administration.rst:402]: OK

Comment (mzorai) [reply docs/Administration.rst:402]: done

**Review by ahamdi — APPROVED** [review]

**Pull request merged**